<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossCorrelationInputUI</class>
 <widget class="QFrame" name="CrossCorrelationInputUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fixed Image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Moving Image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="movingImageFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="fixedImageFile"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="fixedImageButton">
       <property name="text">
        <string>Select...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="movingImageButton">
       <property name="text">
        <string>Select...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Output Image</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="outputImageFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="outputImageButton">
       <property name="text">
        <string>Select...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="processFolder">
     <property name="text">
      <string>Process Folder</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Input Folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output Folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="outputDirectoryLE">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="sourceDirectoryLE"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="sourceDirectoryBtn">
       <property name="text">
        <string>Input Folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="outputDirectoryBtn">
       <property name="text">
        <string>Output Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output Prefix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputPrefix"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output Suffix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputSuffix"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_6">
     <item>
      <widget class="QLabel" name="outputImageTypeLabel">
       <property name="text">
        <string>Output Image Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="outputImageType">
       <item>
        <property name="text">
         <string>tif</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpeg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <widget class="QLabel" name="filterPatternLabel">
       <property name="text">
        <string>File Name Filter: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterPatternLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="fileListView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
